bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/internal/discovery/InboxResource.java

```diff
@@ -76,11 +76,11 @@
     private Inbox inbox;
 
     @Reference(cardinality = ReferenceCardinality.OPTIONAL, policy = ReferencePolicy.DYNAMIC)
-    protected void setInbox(Inbox inbox) {
+    public void setInbox(Inbox inbox) {
```

Comment:
why do we need this change?

Reply:
It seems to work just fine today with a protected modifier. Though earlier in Eclipse it also reported visibility compiler errors here. Maybe this is due to some kind of bug in Eclipse. :disappointed: